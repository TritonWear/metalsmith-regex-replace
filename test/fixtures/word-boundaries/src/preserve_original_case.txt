It doesn't matter if it's a Master Ninja or a rookie ninja.