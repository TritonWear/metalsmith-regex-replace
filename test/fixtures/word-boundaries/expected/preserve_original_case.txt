It doesn't matter if it's a Master Blarg or a rookie blarg.